packages/kununu-utils/newrelic/getConfig/index.js

```diff
@@ -35,6 +35,14 @@ module.exports = () => ({
   distributed_tracing: {
     enabled: 'true',
   },
+  /**
+   * Sets Infinite tracer host if available, see https://one.eu.newrelic.com/edge
+   */
+  infinite_tracing: {
+    trace_observer: {
+      host: (process.env.NR_INFINITE_TRACER_HOST) ? process.env.NR_INFINITE_TRACER_HOST : '',
```

Comment:
I think you could do this:
process.env.NR_INFINITE_TRACER_HOST ?? '',

would be a little cleaner :)

Reply:
Reminder/Info: As seen, it breaks tests as its unsupported language features, so wee need to go with as is